docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -7,17 +7,27 @@ The main media controller features include:
 
 -   Updating and retrieving information
 
-    You can [update the metadata and playback information](#get_media) on the server side, and then retrieve that information on the client side.
+    You can [update the metadata and playback information](#updating-and-retrieving-information) on the server side, and then retrieve that information on the client side.
 
     The media control server provides current information about the registered application that you can send to the client.
 
     When the client requests the information, the media control server updates the state information of an active application before transferring the data. If the application is not running when the client request arrives, the media control server transfers the latest information.
 
+- Updating and Retrieving Playlist
+
+    You can [create a playlist](#updating-and-retrieving-playlist) on the server side, and then retrieve that information on the client side.
+
 - Sending and processing commands
 
-    You can [send a command](#send_media) to the server from the client side, and then process the command on the server side.
+    You can [send a command](#sending-and-processing-commands) on the client side, and then process the command on the server side.
+
+- Setting and Getting Playback Capability
+
+    You can [set the playback capability](#setting-and-getting-playback-capability) on the server side, and then get that capability on the client side.
+
+ - Media Control Metadata Properties
 
-    The client can request server state or [metadata](#servermetadata) information from the server, and receive it through an event handler.
+     You can [set the metadata](#media-control-metadata-properties) on the server side, and then get that metadat on the client side.
```

Comment:
metadata

Reply:
I'll fix it with another PR.

Reply:
Thanks in advance for that. 

On Tizen Developer Site, I updated as "metadata" in advance. You'd possible to see on https://developer.tizen.org/development/guides/.net-application/media-and-camera/media-controller.

Reply:
OK thank you. I'll raise new PR including this typo.